src/Compilers/CSharp/Portable/Symbols/OverriddenOrHiddenMembersResult.cs

```diff
@@ -52,6 +53,37 @@ internal sealed class OverriddenOrHiddenMembersResult
             }
         }
 
+        internal static Symbol GetOverriddenMember(Symbol substitutedOverridingMember, Symbol overriddenByDefinitionMember)
```

Comment:
It feels like what this method actually does is find the version of ```overriddenByDefinitionMember``` in the corresponding base class of ```substitutedOverridingMember```.  I think it might be clearer to express it that way: ```FindCorrespondingBaseTypeMember(Symbol member, NamedTypeSymbol type)```.